third_party/move/move-compiler-v2/tests/bytecode-generator/wildcard1.exp

```diff
@@ -0,0 +1,7 @@
+
+Diagnostics:
+error: the left-hand side has 2 items but the right-hand side provided 1
```

Comment:
Other occurrences of this error message have the correct direction.  Why does this one seem backwards?  Let's fix it here.

Reply:
Also, V1 shows the type of tup() here, which is useful:
```
error[E04005]: expected a single type
  ┌─ tests/move_check/v2/wildcard1.move:7:13
  │
2 │     fun tup(): (u64, u64) {
  │                ---------- Expected a single type, but found expression list type: '(u64, u64)'
  ·
7 │         let _ = tup();
  │             ^ Invalid type for local

```
Please make sure we have a bug filed to improve these sorts of error messages.

Reply:
> Why does this one seem backwards? Let's fix it here.

It was a pre-existing bug in the exp builder, fixed it.

Nice catch.

Reply:
> Please make sure we have a bug filed to improve these sorts of error messages.

There was another issue you filed that was close enough to this one, I added this test case to that issue (https://github.com/aptos-labs/aptos-core/issues/12669).